CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ else()
   set(VERSION_FULL "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
 endif()
 
-message(STATUS "Building eosio.contracts v${VERSION_FULL}")
+message(STATUS "Building eos-system-contracts v${VERSION_FULL}")
```

Comment:
can we also rename this occurrence of eosio?

https://github.com/eosnetworkfoundation/eos-system-contracts/blob/47099915d01ed1028e9e3d4edde38fd882449002/CMakeLists.txt#L3

Reply:
Good point. It's probably fine to rename the project name as well, but I rather leave that up to the devs. There are other implications, like the creation of environment variables with the new name - see https://cmake.org/cmake/help/latest/command/project.html

Reply:
Also, there are other references to github.com/eosio/eosio.contracts in the source code which should be updated too, so the project renaming can be included in that effort later.